docs/data/joy/components/aspect-ratio/FlexRowRatio.js

```diff
@@ -18,10 +18,18 @@ export default function FlexRowRatio() {
         sx={{ gap: 2, minWidth: 300 }}
       >
         <AspectRatio
-          sx={{
-            flexBasis: flexBasis ? `${flexBasis}px` : undefined,
-            overflow: 'auto',
-          }}
+          sx={[
+            {
+              overflow: 'auto',
+            },
+            flexBasis
+              ? {
+                  flexBasis: `${flexBasis}px`,
+                }
+              : {
+                  flexBasis: null,
```

Comment:
```suggestion
                  flexBasis: undefined,
```

If this was done by a codemod I wonder why the value was changed.

Reply:
Pigment sx only works with `null`.